[20/Apr/2018 13:14:31] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:14:35] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:14:38] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:14:41] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:14:43] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:14:44] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:14:46] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:14:48] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:14:51] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:14:54] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:14:56] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:16:22] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:16:35] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:16:35] "GET /api/production/ HTTP/1.0" 200 175179
Not Found: /favicon.ico
[20/Apr/2018 13:16:38] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 13:16:44] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 13:16:46] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 13:16:49] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:16:50] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:16:52] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:16:53] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 13:16:56] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 13:16:57] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:17:18] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:17:18] "GET /api/production/ HTTP/1.0" 200 175179
Not Found: /favicon.ico
[20/Apr/2018 13:17:19] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 13:17:24] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:17:26] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 13:17:27] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:17:33] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:17:35] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:18:13] "GET /api/production/?item=ec2 HTTP/1.0" 200 175179
[20/Apr/2018 13:18:15] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:18:18] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 13:18:20] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 13:18:21] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:18:24] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:18:28] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:18:29] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 13:18:31] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:19:06] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:19:07] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 13:19:09] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:19:24] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:23:01] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:23:01] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:23:17] "GET /api/development HTTP/1.0" 301 0
[20/Apr/2018 13:23:18] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 13:23:19] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:23:23] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:23:25] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:23:47] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 13:23:49] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:23:55] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:23:58] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:25:38] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:25:40] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:25:41] "GET /api/production/?item=ec2 HTTP/1.0" 200 175179
[20/Apr/2018 13:25:43] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 13:25:43] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 13:25:45] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:25:46] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:27:53] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:27:54] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 13:27:56] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 13:27:56] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 13:27:57] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:27:58] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:27:58] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:28:01] "GET /api/production/?item=ec2 HTTP/1.0" 200 175179
[20/Apr/2018 13:28:02] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:28:03] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:33:15] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:33:16] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:33:17] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:33:31] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 13:52:46] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:52:53] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:53:05] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:53:05] "GET /api/production/ HTTP/1.0" 200 175179
Not Found: /favicon.ico
[20/Apr/2018 13:53:06] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 13:53:11] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:53:11] "GET /api/production/ HTTP/1.0" 200 175179
Not Found: /favicon.ico
[20/Apr/2018 13:53:12] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 13:53:28] "GET / HTTP/1.0" 301 0
[20/Apr/2018 13:53:28] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:53:29] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:53:36] "GET /api/production/?item=ec2 HTTP/1.0" 200 175179
[20/Apr/2018 13:53:40] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 13:53:47] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:53:52] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:53:52] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:53:56] "GET /api/development HTTP/1.0" 301 0
[20/Apr/2018 13:53:56] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:01] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:54:02] "GET /api/legacy HTTP/1.0" 301 0
[20/Apr/2018 13:54:02] "GET /api/legacy/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:05] "GET /api/staging HTTP/1.0" 301 0
[20/Apr/2018 13:54:05] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:06] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 13:54:07] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:11] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:54:14] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 13:54:19] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 13:54:25] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 13:54:32] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 13:54:34] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 13:54:36] "GET /api/development HTTP/1.0" 301 0
[20/Apr/2018 13:54:36] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:40] "GET /api/legacy HTTP/1.0" 301 0
[20/Apr/2018 13:54:40] "GET /api/legacy/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:41] "GET /api/staging HTTP/1.0" 301 0
[20/Apr/2018 13:54:41] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 13:54:47] "GET /api/production/ HTTP/1.0" 200 175179
[20/Apr/2018 14:09:44] "GET / HTTP/1.0" 301 0
[20/Apr/2018 14:09:44] "GET /api/production/ HTTP/1.0" 200 173970
Not Found: /favicon.ico
[20/Apr/2018 14:09:45] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 14:10:20] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 14:10:43] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 14:10:51] "GET /api/development HTTP/1.0" 301 0
[20/Apr/2018 14:10:51] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 14:10:53] "GET /api/production/ HTTP/1.0" 200 173970
[20/Apr/2018 14:10:54] "GET /api/legacy HTTP/1.0" 301 0
[20/Apr/2018 14:10:54] "GET /api/legacy/ HTTP/1.0" 200 8038
[20/Apr/2018 14:10:55] "GET /api/staging HTTP/1.0" 301 0
[20/Apr/2018 14:10:55] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 14:10:56] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 14:10:58] "GET /api/production/ HTTP/1.0" 200 173970
[20/Apr/2018 14:13:12] "GET / HTTP/1.0" 301 0
[20/Apr/2018 14:13:13] "GET /api/production/ HTTP/1.0" 200 173970
Not Found: /favicon.ico
[20/Apr/2018 14:13:15] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 14:13:48] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 14:13:58] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 14:14:05] "GET /api/production/?item=ec2 HTTP/1.0" 200 173970
[20/Apr/2018 14:14:08] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 14:14:31] "GET /api/development HTTP/1.0" 301 0
[20/Apr/2018 14:14:32] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 14:14:35] "GET /api/production/?item=ec2 HTTP/1.0" 200 173970
[20/Apr/2018 14:14:41] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 14:14:46] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 14:15:01] "GET /api/staging HTTP/1.0" 301 0
[20/Apr/2018 14:15:02] "GET /api/staging/ HTTP/1.0" 200 8038
[20/Apr/2018 14:15:04] "GET /api/development/ HTTP/1.0" 200 8038
[20/Apr/2018 14:58:07] "GET /api/production/ HTTP/1.0" 200 173970
[20/Apr/2018 14:58:08] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
Not Found: /favicon.ico
[20/Apr/2018 14:58:08] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 14:58:09] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 14:58:09] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 14:58:10] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[20/Apr/2018 14:58:11] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 14:58:15] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 14:58:17] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 14:58:19] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 16:11:21] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
Not Found: /favicon.ico
[20/Apr/2018 16:11:22] "GET /favicon.ico HTTP/1.0" 404 2543
[20/Apr/2018 16:11:25] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 16:11:29] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 16:24:31] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 16:24:32] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 16:24:35] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 16:24:36] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 16:24:38] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 16:24:39] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 16:25:40] "GET /api/production/?item=ec2 HTTP/1.0" 200 171415
[20/Apr/2018 16:25:41] "GET /api/production/ HTTP/1.0" 200 171415
[20/Apr/2018 16:25:44] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 16:25:46] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 16:25:52] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 16:25:54] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 16:25:55] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 16:25:56] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 16:26:04] "GET /api/production/ HTTP/1.0" 200 171415
[20/Apr/2018 16:44:05] "GET /api/production/ HTTP/1.0" 200 171415
[20/Apr/2018 16:44:23] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 16:44:25] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 16:44:27] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 16:44:28] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 16:44:30] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 16:44:31] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 16:44:32] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 16:44:34] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 16:44:37] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 16:44:38] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 16:44:59] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 16:45:01] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[20/Apr/2018 16:45:02] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[20/Apr/2018 16:45:04] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 16:45:06] "GET /api/production/?item=route53&id=Z1HPEFOO7APF4D HTTP/1.0" 200 11544
[20/Apr/2018 16:45:08] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 16:45:11] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 16:46:01] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[20/Apr/2018 17:50:49] "GET /api/production/ HTTP/1.0" 200 171415
[20/Apr/2018 17:50:51] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 17:50:53] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 17:51:09] "GET /api/production/?item=ec2 HTTP/1.0" 200 171415
[20/Apr/2018 17:51:34] "GET /api/production/?item=ebs HTTP/1.0" 200 37564
[20/Apr/2018 17:51:38] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12254
[20/Apr/2018 17:51:40] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 17281
[20/Apr/2018 17:51:43] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[20/Apr/2018 17:57:40] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[20/Apr/2018 17:57:49] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[23/Apr/2018 09:23:37] "GET /api/production/ HTTP/1.0" 200 177221
Not Found: /favicon.ico
[23/Apr/2018 09:23:38] "GET /favicon.ico HTTP/1.0" 404 2543
[23/Apr/2018 09:28:31] "GET /api/production/ HTTP/1.0" 200 177221
[23/Apr/2018 09:28:33] "GET /api/legacy/ HTTP/1.0" 200 8038
[23/Apr/2018 09:28:37] "GET /api/production/ HTTP/1.0" 200 177221
[23/Apr/2018 09:28:39] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[23/Apr/2018 11:40:36] "GET /api/production/ HTTP/1.0" 200 176246
[23/Apr/2018 12:28:13] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[23/Apr/2018 15:14:23] "GET /api/development HTTP/1.0" 301 0
[23/Apr/2018 15:14:23] "GET /api/development/ HTTP/1.0" 200 8038
[23/Apr/2018 15:14:25] "GET /api/production/ HTTP/1.0" 200 172535
[23/Apr/2018 15:14:26] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[23/Apr/2018 15:14:27] "GET /api/production/?item=ebs HTTP/1.0" 200 37584
[23/Apr/2018 15:14:28] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[23/Apr/2018 15:14:30] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[23/Apr/2018 15:14:31] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[23/Apr/2018 15:17:21] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
Not Found: /favicon.ico
[23/Apr/2018 15:17:21] "GET /favicon.ico HTTP/1.0" 404 2543
[23/Apr/2018 15:18:58] "GET /api/production/?item=route53&id=Z3AOFP6XZFI8VU HTTP/1.0" 200 9170
[23/Apr/2018 15:19:05] "GET /api/development/ HTTP/1.0" 200 8038
[23/Apr/2018 15:19:06] "GET /api/production/ HTTP/1.0" 200 172535
[23/Apr/2018 15:19:07] "GET / HTTP/1.0" 301 0
[23/Apr/2018 15:19:08] "GET /api/production/ HTTP/1.0" 200 172535
[23/Apr/2018 15:19:46] "GET /api/production/?item=ebs HTTP/1.0" 200 37584
[23/Apr/2018 15:19:48] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[23/Apr/2018 15:19:51] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10305
[24/Apr/2018 09:49:38] "GET /api/production/ HTTP/1.0" 200 186079
[25/Apr/2018 20:24:05] "GET /api/production/ HTTP/1.0" 200 173499
Not Found: /favicon.ico
[25/Apr/2018 20:24:06] "GET /favicon.ico HTTP/1.0" 404 2543
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 81, in get_production_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.production_Zone_[grindrstore.com.]_Id_[Z1S6I48EJQF6W6]' doesn't exist")
[25/Apr/2018 20:24:16] "GET /api/production/?item=route53&id=Z1S6I48EJQF6W6 HTTP/1.0" 500 109149
[25/Apr/2018 20:24:31] "GET /api/development HTTP/1.0" 301 0
[25/Apr/2018 20:24:31] "GET /api/development/ HTTP/1.0" 200 8038
[25/Apr/2018 20:24:36] "GET /api/production/?item=ebs HTTP/1.0" 200 40211
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 20, 2018 - 13:14:13
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
[25/Apr/2018 20:28:29] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:28:30] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:28:32] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:28:40] "GET /api/production/?item=route53&id=Z1T8M215NT314 HTTP/1.0" 200 19951
[25/Apr/2018 20:28:41] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 14924
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 81, in get_production_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.production_Zone_[grindrguy.net.]_Id_[Z1IEX3AJHVNIMO]' doesn't exist")
[25/Apr/2018 20:28:43] "GET /api/production/?item=route53&id=Z1IEX3AJHVNIMO HTTP/1.0" 500 109146
[25/Apr/2018 20:28:47] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:28:50] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:07] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:08] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:10] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 14924
[25/Apr/2018 20:29:11] "GET /api/production/?item=ebs HTTP/1.0" 200 40211
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 81, in get_production_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.production_Zone_[grindrguy.net.]_Id_[Z1IEX3AJHVNIMO]' doesn't exist")
[25/Apr/2018 20:29:13] "GET /api/production/?item=route53&id=Z1IEX3AJHVNIMO HTTP/1.0" 500 109120
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 81, in get_production_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.production_Zone_[stg.grindr.io.]_Id_[Z2HTQVO8VCSEGD]' doesn't exist")
[25/Apr/2018 20:29:38] "GET /api/production/?item=route53&id=Z2HTQVO8VCSEGD HTTP/1.0" 500 109120
[25/Apr/2018 20:29:47] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:49] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:50] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:50] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:52] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:29:54] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:30:10] "GET /api/production/ HTTP/1.0" 200 173499
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 81, in get_production_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.production_Zone_[grindr.com.]_Id_[Z2RCOV37KYHCDD]' doesn't exist")
[25/Apr/2018 20:30:16] "GET /api/production/?item=route53&id=Z2RCOV37KYHCDD HTTP/1.0" 500 108682
[25/Apr/2018 20:30:18] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:30:20] "GET /api/production/?item=ebs HTTP/1.0" 200 40211
[25/Apr/2018 20:30:34] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:30:36] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:30:38] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:30:40] "GET /api/production/?item=s3 HTTP/1.0" 200 48089
[25/Apr/2018 20:30:44] "GET /api/production/?item=ec2 HTTP/1.0" 200 173499
[25/Apr/2018 20:31:31] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:31:33] "GET /api/production/ HTTP/1.0" 200 173499
[25/Apr/2018 20:31:34] "GET /api/production/ HTTP/1.0" 200 173499
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 25, 2018 - 20:28:26
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
[25/Apr/2018 20:33:13] "GET /api/production/ HTTP/1.0" 200 170829
[25/Apr/2018 20:33:15] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[25/Apr/2018 20:33:17] "GET /api/development/ HTTP/1.0" 200 8038
[25/Apr/2018 20:33:19] "GET /api/production/ HTTP/1.0" 200 170829
[25/Apr/2018 20:33:21] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[25/Apr/2018 20:34:40] "GET /api/development/ HTTP/1.0" 200 8038
[25/Apr/2018 20:35:06] "GET /api/production/ HTTP/1.0" 200 170829
[25/Apr/2018 20:35:08] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
[25/Apr/2018 20:35:10] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67378
[25/Apr/2018 20:35:12] "GET /api/production/?item=ec2 HTTP/1.0" 200 170829
[25/Apr/2018 20:35:14] "GET /api/production/?item=s3 HTTP/1.0" 200 45419
[25/Apr/2018 20:35:15] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13631
[26/Apr/2018 09:07:03] "GET /api/production/ HTTP/1.0" 200 176866
[26/Apr/2018 09:07:06] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14664
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 25, 2018 - 20:33:10
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 71, in get_production_data
    return render(request, 'table.html', content1)
  File "/usr/lib64/python2.7/site-packages/django/shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/usr/lib64/python2.7/site-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 68, in render
    reraise(exc, self.backend)
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 89, in reraise
    six.reraise(exc.__class__, new, sys.exc_info()[2])
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 154, in render
    compiled_parent = self.get_parent(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 151, in get_parent
    return self.find_template(parent, context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 131, in find_template
    template_name, skip=history,
  File "/usr/lib64/python2.7/site-packages/django/template/engine.py", line 148, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
TemplateDoesNotExist: {{ env }}.html
[26/Apr/2018 10:04:26] "GET /api/production/ HTTP/1.0" 500 201844
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 71, in get_production_data
    return render(request, 'table.html', content1)
  File "/usr/lib64/python2.7/site-packages/django/shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/usr/lib64/python2.7/site-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 68, in render
    reraise(exc, self.backend)
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 89, in reraise
    six.reraise(exc.__class__, new, sys.exc_info()[2])
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 204, in render
    template = context.template.engine.get_template(template_name)
  File "/usr/lib64/python2.7/site-packages/django/template/engine.py", line 162, in get_template
    template, origin = self.find_template(template_name)
  File "/usr/lib64/python2.7/site-packages/django/template/engine.py", line 148, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
TemplateDoesNotExist: {{ env }}.sidebar.html
[26/Apr/2018 10:08:52] "GET /api/production/ HTTP/1.0" 500 233372
Internal Server Error: /api/production/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 71, in get_production_data
    return render(request, 'table.html', content1)
  File "/usr/lib64/python2.7/site-packages/django/shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/usr/lib64/python2.7/site-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/usr/lib64/python2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 154, in render
    compiled_parent = self.get_parent(context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 151, in get_parent
    return self.find_template(parent, context)
  File "/usr/lib64/python2.7/site-packages/django/template/loader_tags.py", line 131, in find_template
    template_name, skip=history,
  File "/usr/lib64/python2.7/site-packages/django/template/engine.py", line 136, in find_template
    name, template_dirs=dirs, skip=skip,
  File "/usr/lib64/python2.7/site-packages/django/template/loaders/base.py", line 44, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 191, in __init__
    self.nodelist = self.compile_nodelist()
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 230, in compile_nodelist
    return parser.parse()
  File "/usr/lib64/python2.7/site-packages/django/template/base.py", line 515, in parse
    raise self.error(token, e)
TemplateSyntaxError: Unknown argument for u'include' tag: u'env'.
[26/Apr/2018 10:09:14] "GET /api/production/ HTTP/1.0" 500 213410
[26/Apr/2018 10:15:28] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:15:31] "GET /api/development/ HTTP/1.0" 200 7771
[26/Apr/2018 10:15:35] "GET /api/production/?item=ebs HTTP/1.0" 200 37502
[26/Apr/2018 10:15:36] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14623
[26/Apr/2018 10:15:38] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14623
[26/Apr/2018 10:15:40] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:15:42] "GET /api/development/ HTTP/1.0" 200 7771
[26/Apr/2018 10:15:43] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:15:45] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67337
[26/Apr/2018 10:16:14] "GET /api/production/?item=ebs HTTP/1.0" 200 37502
[26/Apr/2018 10:30:57] "GET /api/development/ HTTP/1.0" 200 7771
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 26, 2018 - 10:03:53
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
[26/Apr/2018 10:34:57] "GET /api/development/ HTTP/1.0" 200 111701
[26/Apr/2018 10:35:00] "GET /api/development/?item=ebs HTTP/1.0" 200 20190
[26/Apr/2018 10:35:08] "GET /api/development/?item=route53&id=Z39K5A0GQLGUSL HTTP/1.0" 200 8840
[26/Apr/2018 10:35:11] "GET /api/development/?item=route53&id=Z2KY29K93KM131 HTTP/1.0" 200 13191
[26/Apr/2018 10:35:13] "GET /api/development/?item=route53&id=ZH2Y70RD294AJ HTTP/1.0" 200 17797
[26/Apr/2018 10:35:24] "GET /api/development/?item=route53&id=Z2XDKA4RUM0S2S HTTP/1.0" 200 25547
[26/Apr/2018 10:35:29] "GET /api/development/?item=ec2 HTTP/1.0" 200 111701
[26/Apr/2018 10:35:30] "GET /api/development/?item=s3 HTTP/1.0" 200 35679
[26/Apr/2018 10:35:39] "GET /api/development/?item=ec2 HTTP/1.0" 200 111701
[26/Apr/2018 10:35:45] "GET /api/development/?item=route53&id=Z2KY29K93KM131 HTTP/1.0" 200 13191
[26/Apr/2018 10:35:46] "GET /api/development/?item=ebs HTTP/1.0" 200 20190
[26/Apr/2018 10:35:48] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:35:49] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:35:52] "GET /api/development/ HTTP/1.0" 200 111701
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 26, 2018 - 10:34:38
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).

You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
April 26, 2018 - 10:39:22
Django version 1.11.12, using settings 'myproject.settings'
Starting development server at http://0.0.0.0:8002/
Quit the server with CONTROL-C.
[26/Apr/2018 10:47:07] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:47:09] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 10:47:10] "GET /api/legacy HTTP/1.0" 301 0
[26/Apr/2018 10:47:11] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 10:47:13] "GET /api/legacy/?item=ebs HTTP/1.0" 200 9516
[26/Apr/2018 10:47:15] "GET /api/legacy/?item=route53&id=Z2ZG9OUDCGOOYR HTTP/1.0" 200 12453
[26/Apr/2018 10:47:17] "GET /api/legacy/?item=route53&id=Z1IEX3AJHVNIMO HTTP/1.0" 200 29197
[26/Apr/2018 10:47:19] "GET /api/legacy/?item=route53&id=Z1RCUXX7W0A26O HTTP/1.0" 200 19045
[26/Apr/2018 10:47:21] "GET /api/legacy/?item=route53&id=Z1RCUXX7W0A26O HTTP/1.0" 200 19045
[26/Apr/2018 10:47:23] "GET /api/legacy/?item=route53&id=Z93VHC1JV1TYG HTTP/1.0" 200 10842
[26/Apr/2018 10:47:27] "GET /api/legacy/?item=route53&id=Z3DN439AXT68NJ HTTP/1.0" 200 17680
[26/Apr/2018 10:47:28] "GET /api/legacy/?item=route53&id=Z3C2HD2USPWN8J HTTP/1.0" 200 10948
[26/Apr/2018 10:47:29] "GET /api/legacy/?item=route53&id=Z3AOLNMK5LB0LL HTTP/1.0" 200 13250
[26/Apr/2018 10:47:30] "GET /api/legacy/?item=route53&id=Z1S6I48EJQF6W6 HTTP/1.0" 200 11753
[26/Apr/2018 10:47:32] "GET /api/staging HTTP/1.0" 301 0
[26/Apr/2018 10:47:33] "GET /api/staging/ HTTP/1.0" 200 8692
[26/Apr/2018 10:47:36] "GET /api/staging/?item=ec2 HTTP/1.0" 200 8692
[26/Apr/2018 10:47:37] "GET /api/staging/?item=s3 HTTP/1.0" 200 21504
[26/Apr/2018 10:47:48] "GET /api/legacy/ HTTP/1.0" 200 26930
Internal Server Error: /api/legacy/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 299, in get_legacy_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.legacy_S3' doesn't exist")
[26/Apr/2018 10:47:50] "GET /api/legacy/?item=s3 HTTP/1.0" 500 105522
[26/Apr/2018 10:48:08] "GET /api/legacy/?item=ebs HTTP/1.0" 200 9516
[26/Apr/2018 10:48:09] "GET /api/legacy/?item=route53&id=Z2ZG9OUDCGOOYR HTTP/1.0" 200 12453
[26/Apr/2018 10:48:10] "GET /api/legacy/?item=route53&id=Z1IEX3AJHVNIMO HTTP/1.0" 200 29197
[26/Apr/2018 10:48:16] "GET /api/legacy/?item=route53&id=Z93VHC1JV1TYG HTTP/1.0" 200 10842
[26/Apr/2018 10:48:17] "GET /api/legacy/?item=route53&id=Z3VQ2VM3NEVS8P HTTP/1.0" 200 9634
[26/Apr/2018 10:48:18] "GET /api/legacy/?item=route53&id=Z2XCSGKETBUXPN HTTP/1.0" 200 9625
[26/Apr/2018 10:48:18] "GET /api/legacy/?item=route53&id=Z3DN439AXT68NJ HTTP/1.0" 200 17680
[26/Apr/2018 10:48:20] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:48:21] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 10:48:23] "GET /api/production/?item=ebs HTTP/1.0" 200 37502
[26/Apr/2018 10:48:30] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12213
[26/Apr/2018 10:48:31] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67337
[26/Apr/2018 10:48:32] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:00:54] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:00:57] "GET /api/legacy/?item=route53&id=Z1RCUXX7W0A26O HTTP/1.0" 200 19045
[26/Apr/2018 11:00:57] "GET /api/legacy/?item=route53&id=Z93VHC1JV1TYG HTTP/1.0" 200 10842
[26/Apr/2018 11:00:59] "GET /api/legacy/?item=route53&id=Z2XCSGKETBUXPN HTTP/1.0" 200 9625
[26/Apr/2018 11:01:02] "GET /api/legacy/?item=route53&id=Z3DN439AXT68NJ HTTP/1.0" 200 17680
[26/Apr/2018 11:01:16] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:01:17] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:01:19] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14623
[26/Apr/2018 11:01:21] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:01:23] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:01:24] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:01:26] "GET /api/development/?item=route53&id=Z39K5A0GQLGUSL HTTP/1.0" 200 8854
[26/Apr/2018 11:01:27] "GET /api/development/?item=route53&id=Z2KY29K93KM131 HTTP/1.0" 200 13205
[26/Apr/2018 11:01:29] "GET /api/development/?item=route53&id=ZH2Y70RD294AJ HTTP/1.0" 200 17811
[26/Apr/2018 11:02:22] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:02:23] "GET /api/production/?item=route53&id=Z38JFD4LMHKUJP HTTP/1.0" 200 12213
[26/Apr/2018 11:02:24] "GET /api/production/?item=route53&id=Z2SOFZBB1QKURC HTTP/1.0" 200 67337
[26/Apr/2018 11:02:25] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10264
[26/Apr/2018 11:02:26] "GET /api/production/?item=route53&id=Z380AXUJ5LZJQ5 HTTP/1.0" 200 13590
[26/Apr/2018 11:02:30] "GET /api/production/?item=route53&id=ZOTV85W1CYOWB HTTP/1.0" 200 10264
[26/Apr/2018 11:02:31] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:02:33] "GET /api/development/?item=route53&id=ZH2Y70RD294AJ HTTP/1.0" 200 17811
[26/Apr/2018 11:02:34] "GET /api/development/?item=route53&id=Z39K5A0GQLGUSL HTTP/1.0" 200 8854
[26/Apr/2018 11:02:35] "GET /api/development/?item=ebs HTTP/1.0" 200 20204
[26/Apr/2018 11:02:40] "GET /api/staging/ HTTP/1.0" 200 8692
[26/Apr/2018 11:02:42] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:02:43] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:02:44] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:02:45] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:02:48] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:02:49] "GET /api/staging/ HTTP/1.0" 200 8692
[26/Apr/2018 11:02:51] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:02:53] "GET /api/production/?item=route53&id=Z2R2BOBPMOS28B HTTP/1.0" 200 14623
[26/Apr/2018 11:02:55] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:02:56] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:03:04] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:03:05] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:35] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:05:36] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:38] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:40] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:05:50] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:51] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:05:53] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:54] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:05:56] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:05:57] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:58] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:05:59] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:06:00] "GET /api/development/ HTTP/1.0" 200 111715
[26/Apr/2018 11:06:01] "GET /api/production/ HTTP/1.0" 200 176825
[26/Apr/2018 11:06:24] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:06:26] "GET /api/legacy/?item=route53&id=Z2ZG9OUDCGOOYR HTTP/1.0" 200 12453
[26/Apr/2018 11:06:58] "GET /api/legacy/ HTTP/1.0" 200 26930
[26/Apr/2018 11:06:59] "GET /api/legacy/?item=route53&id=Z1RCUXX7W0A26O HTTP/1.0" 200 19045
[26/Apr/2018 11:07:01] "GET /api/legacy/?item=ebs HTTP/1.0" 200 9516
Internal Server Error: /api/legacy/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/myproject/myapp/views.py", line 299, in get_legacy_data
    for i in connection.introspection.get_table_description(cursor, table_name):
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/introspection.py", line 76, in get_table_description
    cursor.execute("SELECT * FROM %s LIMIT 1" % self.connection.ops.quote_name(table_name))
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib64/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
  File "/usr/lib64/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib64/python2.7/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1146, "Table 'ec2_test.aws.legacy_S3' doesn't exist")
[26/Apr/2018 11:07:02] "GET /api/legacy/?item=s3 HTTP/1.0" 500 105531
[26/Apr/2018 11:07:07] "GET /api/legacy/?item=route53&id=Z3VQ2VM3NEVS8P HTTP/1.0" 200 9634
[26/Apr/2018 11:07:08] "GET /api/legacy/?item=route53&id=Z2XCSGKETBUXPN HTTP/1.0" 200 9625
[26/Apr/2018 11:07:10] "GET /api/legacy/?item=route53&id=Z2RCOV37KYHCDD HTTP/1.0" 200 72306
[26/Apr/2018 11:07:11] "GET /api/legacy/?item=route53&id=Z3C2HD2USPWN8J HTTP/1.0" 200 10948
[26/Apr/2018 11:07:18] "GET /api/legacy/?item=route53&id=Z3AOLNMK5LB0LL HTTP/1.0" 200 13250
[26/Apr/2018 11:07:18] "GET /api/legacy/?item=route53&id=Z1S6I48EJQF6W6 HTTP/1.0" 200 11753
[26/Apr/2018 11:07:20] "GET /api/legacy/?item=route53&id=ZSWJ644GMWU7J HTTP/1.0" 200 173266
[26/Apr/2018 11:07:21] "GET /api/legacy/?item=route53&id=Z38HN3MBVVY22K HTTP/1.0" 200 9634
[26/Apr/2018 11:07:22] "GET /api/legacy/?item=route53&id=Z2UF7HL2T4QCX2 HTTP/1.0" 200 9629
[26/Apr/2018 11:07:25] "GET /api/legacy/ HTTP/1.0" 200 26930
